version

```diff
@@ -1 +1 @@
-1.9.1
\ No newline at end of file
+1.9.2
```

Comment:
Please do not uptick the version in PRs. See https://github.com/egnyte/gitlabform/commit/87819c313f36748d988fe3a08b862da4112a4661 for a rationale.